mappings.py

```diff
@@ -1,4 +1,5 @@
 color_map = {
+    "#452A00": 0,  # transparent
```

Comment:
Won't this make closest_color sometimes choose transparent as a color ?

Reply:
yes. it should only be added to the name map